src/commons/BrowserUtils.ts

```diff
@@ -777,10 +777,15 @@ export namespace BrowserUtils {
   /**
    * Send a sequence of key strokes to the active element
    * it can be single key or an array of keys
-   * @param keysToSend key or array of keys to send
+   * @param keysToSend key, array of keys or string array (chars) to send
    */
-  export function sendKeys(keysToSend: SpecialKeys | SpecialKeys[]): void {
-    Reporter.debug(`Sending Keys ${getKeyNames(keysToSend)}`);
+  export function sendKeys(keysToSend: SpecialKeys | SpecialKeys[] | string[]): void {
```

Comment:
why do you use string[] ? what benefits does it have over string ?

Reply:
Don't know. the example presented as a solution was done using string[]

Reply:
Changed type to string and splice inside sendKeys to avoid saved word such as 'Enter', 'F5' etc'